### DISCUSSION TITLE
Sticker kadang eror

### DISCUSSION THREAD

**Discussion by Patrick12a** [Nurutomo/wabot-aq#943] [Q&A]
mohon  maaf, Saat saya mengunggah foto / me reply pesan foto pasti terjadi error : Conversation Failed, itu kenapa yah, padahal untuk video ke sticker aman2 aja, mohon bantuannya suhu

**Comment by unx21** [comment]
> mohon maaf, Saat saya mengunggah foto / me reply pesan foto pasti terjadi error : Conversation Failed, itu kenapa yah, padahal untuk video ke sticker aman2 aja, mohon bantuannya 

#940
